src/main/java/seedu/address/logic/commands/student/ExtractEmailsCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands.student;
+
+import java.util.stream.Collectors;
+
+import javafx.collections.ObservableList;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.student.Student;
+
+
+/**
+ * Shows student's grade distribution in ModQuik.
+ */
+public class ExtractEmailsCommand extends Command {
+
+    public static final String COMMAND_WORD = "extract emails";
+
+    public static final String MESSAGE_SUCCESS = "Copied link to clipboard! Open it in a browser.";
+
+    @Override
+    public CommandResult execute(Model model) {
```

Comment:
If you want, I suggest adding a simple test for this command, to be safe, I think it can just test the success message, which is similar to the diagram test.